functions/linear_algebra/triangular.stan

```diff
@@ -7,18 +7,16 @@
    K is N * N / 2
    * @param vector of lower triangular elements
    */
-vector lower_ele (matrix mat, int K){
-  int N = rows(mat);
-  vector[K] out;
-  
-  int count = N;
-  int pos = 1;
-    
-  for (k in 1:K - 1){
-    count -= 1;
-    out[pos:pos + count - 1] = sub_col(mat, k + 1, k, count);
-    pos += count;
-  }
-  
-  return out;
+  vector lower_elements(matrix M, int tri_size){
```

Comment:
Does the previous version not work? I have some vague remembrance of a potential issue here.